Text/us 402 (customizing workflow-landda) (#1)
gspetro-NOAA/land-DA_workflow: doc/source/CustomizingTheWorkflow/ConfigWorkflow.rst

``NPROCS_ANALYSIS:`` (Default: "6")
   Number of processors for the analysis task. 

``DT_ATMOS:`` (Default: "900")

>    Time integration step of the atmospheric component of the UFS weather model (in seconds).

Reviewer: ```suggestion
   The main integration time step of the atmospheric component of the UFS Weather Model (in seconds). This is the time step for the outermost atmospheric model loop and must be a positive integer value. It corresponds to the frequency at which the physics routines and the top level dynamics routine are called. (Note that one call to the top-level dynamics routine results in multiple calls to the horizontal dynamics, tracer transport, and vertical dynamics routines; see the `FV3 dycore scientific documentation <https://repository.library.noaa.gov/view/noaa/30725>`_ for details.) 
```

   
``DT_RUNSEQ:`` (Default: "6")

>    Time interval of run sequence (coupling interval) between the model components of the UFS weather model (in seconds).

Reviewer: ```suggestion
   Time interval of run sequence (coupling interval) between the model components of the UFS Weather Model (in seconds).
```

``NPROCS_FORECAST:`` (Default: "26")
   Total number of processes for the FORECAST task.

``NPROCS_FORECAST_ATM:`` (Default: "12")
   Number of processes for the atmospheric model component (DATM) in the FORECAST task.

``NPROCS_FORECAST_LND:`` (Default: "12")

>    Number of processes for the land model component (Noah-MP) in the FORECAST task.

Reviewer: Any idea why these only add up to 24 instead of the 26 listed in `NPROCS_FORECAST`? I'm curious how they're calculated...

``DAtype:`` (Default: "letkfoi_snow")
   Type of data assimilation. Valid values: ``letkfoi_snow``. Currently, Land DA only performs snow DA using the LETKF-OI algorithm. As the application expands, more options may be added. 

``TSTUB:`` (Default: "oro_C96.mx100")
   Specifies the file stub/name for orography files in ``TPATH``. This file stub is named ``oro_C${RES}`` for atmosphere-only orography files and ``oro_C{RES}.mx100`` for atmosphere and ocean orography files. When Land DA is compiled with ``sorc/app_build.sh``, the subdirectories of the fix files should be linked into the ``fix`` directory, and orography files can be found in ``fix/FV3_fix_tiled/C96``. 

``WE2E_VAV:`` (Default: "YES")

>    Flag of the workflow end-to-end (WE2E) test for the verification and validation ("YES"/"NO")

Reviewer: ```suggestion
   Flag to turn on the workflow end-to-end (WE2E) test. When WE2E_VAV="YES", the result files from the experiment are compared to the test baseline files, located in ``fix/test_base/we2e_com``. If the results are within the tolerance set (via ``WE2E_ATOL``) at the end of the three main tasks --- ``analysis``, ``forecast``, and ``post_anal`` --- then the experiment passes. Valid values: ``"YES"`` | ``"NO"``
```

``WE2E_ATOL:`` (Default: "1e-7")
   Tolerance of the WE2E test

``WE2E_LOG_FN:`` (Default: "we2e.log")

>    Log file name of the WE2E test

Reviewer: ```suggestion
   Name of the WE2E test log file
```
